app/controllers/providers/provider_base_controller.rb

```diff
@@ -21,5 +21,15 @@ def link_banner_display
 
       I18n.t("providers.submitted_applications.show.link_banner_one")
     end
+
+    def set_transaction_types
```

Comment:
could this be a concern rather than on _all_ provider controllers?  Seems a bit off an overhead!

Reply:
Have updated the last commit to use a concern instead